indico/modules/events/reminders/templates/reminders.html

```diff
@@ -11,7 +11,15 @@
         {% for reminder in reminders %}
             <li>
                 <strong>
-                    <a href="{{ url_for('.edit', reminder) }}">
+                    <a href="#" data-href="{{ url_for('.edit', reminder) }}"
+                       data-ajax-dialog
+                       data-hide-page-header
+                       data-reload-after
+                       data-title="{% if not reminder.is_sent %}
```

Comment:
we usually put the no-argument attributes last, so I'd move this one before `data-ajax-dialog`

it might also be more readable to assign it first and just use `data-title="{{ button_title }}"`:

```
{% if ... %}
    {% set button_title = _("Edit reminder") %}
{% else %}
    {% set button_title = _("Reminder details") %}
{% endif %}
```